app/src/main/res/layout/screen_patients.xml

```diff
@@ -60,7 +60,7 @@
           tools:ignore="ContentDescription" />
 
         <include
-          android:id="@+id/simpleVideoLayout"
+          android:id="@+id/simpleIllustrationLayout"
```

Comment:
Why are we naming `simple_video_layout` as illustration layout? We already have an illustration view. So, we can get rid of this and the video layout?